src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,126 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.library.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static String DB_NAME = "internetshop";
+    private static String TABLE = "items";
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item entity) {
+        String query = String.format("INSERT INTO %s(name, price) VALUES('%s', %f)",
+                TABLE, entity.getName(), entity.getPrice());
+
+        try (Statement stmt = connection.createStatement()) {
+            stmt.executeUpdate(query);
+        } catch (SQLException e) {
+            logger.warn(e);
```

Comment:
```suggestion
            logger.warn("can't do something with this thing " + thing, e);
```
варто додавати власний текст в логер, щоб отримати більш конкретизовану інформацію